packages/mermaid/src/diagrams/sequence/sequenceDb.ts

```diff
@@ -228,13 +228,24 @@ export const setWrap = function (wrapSetting?: boolean) {
   state.records.wrapEnabled = wrapSetting;
 };
 
+const extractWrap = (text?: string): { cleanedText?: string; wrap?: boolean } => {
+  if (!text) {
```

Comment:
It looks like this function acts differently if `text` is an empty string `''`. Is this expected behaviour? If not, we should replace this with:

```suggestion
  if (text === undefined) {
```